erpnext/payroll/doctype/salary_slip/salary_slip.json

```diff
@@ -578,13 +580,27 @@
   {
    "fieldname": "column_break_69",
    "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "year_to_date",
+   "fieldtype": "Currency",
+   "label": "Year To Date(Company Currency)",
```

Comment:
You'll have to format these fields on client side for the proper currency to show up
Also add description for both these fields